client/scripts/controllers/chain/plasm/main.ts

```diff
@@ -6,11 +6,23 @@ import Substrate from '../substrate/main';
 
 class Plasm extends Substrate {
   constructor(n: NodeInfo, app: IApp) {
-    super(n, app, ChainClass.Plasm, false); // disable all governance modules
+    super(n, app, ChainClass.Plasm);
+
+    // disable all governance modules
+    this.phragmenElections.disable();
+    this.council.disable();
+    this.technicalCommittee.disable();
+    this.democracyProposals.disable();
+    this.democracy.disable();
+    this.treasury.disable();
+    this.signaling.disable();
+
+    // TODO: should we disable identity as well?
```

Comment:
Yeah, we should disable identities, otherwise we get an error when trying to log in with Plasm.